userland/examples/sdcard/main.c

```diff
@@ -0,0 +1,101 @@
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <tock.h>
+#include <console.h>
+#include <timer.h>
+#include <sdcard.h>
```

Comment:
I think we are using `"` for referencing `libtock` from within `libtock`, but `<` from outside (e.g. in examples)